packages/validate-icu-locales/README.md

```diff
@@ -0,0 +1,46 @@
+# `@scaleway/validate-icu-locales`
+
+## A tiny cli to handle ICU errors on locales files.
+## Install
+
+```bash
+$ pnpm add -D @scaleway/validate-icu-locales
+```
+## Usage
+
+We can parse JSON, TS and JS if theses files use default export.
+
+
+```
+node @scaleway/validate-icu-locales "../**/en.json"
```

Comment:
why node?

Reply:
add rollup plugin to preserve shebang :)